packages/patrol/example/ios/Runner/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSBonjourServices</key>
```

Comment:
If this app was to be deployed to App Store, it'd be a tad harder.

https://docs.flutter.dev/development/add-to-app/ios/project-setup#local-network-privacy-permissions